contribs/gmf/src/controllers/abstract.js

```diff
@@ -180,4 +181,31 @@ gmf.AbstractController.prototype.initLanguage = function() {
   }
 };
 
+/**
+ * @param {angular.$injector} $injector Main injector.
+ * @return {olx.ViewOptions} The view options from the permalink.
+ * @protected
+ */
+gmf.AbstractController.prototype.getPermalinkViewConfig = function($injector) {
```

Comment:
It would make sense to put the `$injector` in `this` in the abstract class, but the map is created before calling the parent constructor.  I suppose it's okay if it's created after.